app/src/main/java/sopt/motivoo/presentation/home/HomePhotoDialogFragment.kt

```diff
@@ -0,0 +1,49 @@
+package sopt.motivoo.presentation.home
+
+import android.graphics.Bitmap
+import android.graphics.ImageDecoder
+import android.net.Uri
+import android.os.Build
+import android.os.Bundle
+import android.provider.MediaStore
+import android.view.View
+import androidx.navigation.fragment.findNavController
+import androidx.navigation.fragment.navArgs
+import coil.load
+import sopt.motivoo.R
+import sopt.motivoo.databinding.DialogHomePhotoBinding
+import sopt.motivoo.util.binding.BindingDialogFragment
+
+class HomePhotoDialogFragment :
+    BindingDialogFragment<DialogHomePhotoBinding>(R.layout.dialog_home_photo) {
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        setLayoutSize(1f, 1f)
```

Comment:
크.. 부모 클래스에 정의해두셨군요 !!
```suggestion
        setLayoutSize(widthPercent = 1f, heightPercent = 1f)
```

kotlin으로 선언한 함수의 경우 다음과 같이 인자의 이름을 명시해줄 수 있습니다! 다음과 같이 하면 1f의 느낌이 확 와닿을 것 같아요!
아니면 함수의 이름에 ratio를 넣어서 비율을 지정해주는 것을 명시화하는 것도 좋아보입니다 !

Reply:
좋은 생각이십니다..! 해당 함수에서는 함수 파라미터를 명시적으로 보여주는 것이 더 좋은 코드를 작성할 수 있는 부분인 것 같습니다. 
[피드백 수정, 커밋 : 1968e1e](https://github.com/Team-Motivoo/Motivoo-Android/pull/40/commits/1968e1efb27ba6a3f73495a68afba6ae0abc7384)